libbeat/processors/actions/docs/decode_json_fields.asciidoc

```diff
@@ -23,26 +23,26 @@ processors:
 The `decode_json_fields` processor has the following configuration settings:
 
 `fields`:: The fields containing JSON strings to decode.
-`process_array`:: (Optional) A boolean that specifies whether to process
-arrays. The default is false.
-`max_depth`:: (Optional) The maximum parsing depth. A value of 1 will decode the
+`process_array`:: (Optional) A Boolean that specifies whether to process
```

Comment:
I capitalized boolean throughout this, as it was capitalized in an earlier article that I read. Also, should we add value to all of these? I think that's what it should be but wanted to double check before we did that!

Reply:
Yup. Looks like the style guide says to capitalize it, so we should, too. Boolean is one of those odd words that's capitalized because it's derived from a name. I will add this to our cleanup list of common problems to fix. :-)